index.js

```diff
@@ -9,18 +9,18 @@ module.exports = function ({types: t}) {
 
         const ifBinding = getAndRemoveIfBinding(path.node.openingElement);
         if (ifBinding) {
-            path.replaceWith(t.jSXExpressionContainer(
```

Comment:
Actually I was wondering why you used t.jSXExpressionContainer as a wrapper?

Reply:
😂 I just did as what I saw at [AST explorer](https://astexplorer.net/) and it's worked fine for me.